.github/workflows/rust.yml

```diff
@@ -108,11 +108,29 @@ jobs:
           command: clippy
           args: --all-targets --features unstable-light-client -- -D warnings
 
-      - name: Cargo clippy for web
+      - name: Cargo clippy for web subxt-lightclient
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --no-default-features --features web -- -D warnings
+          args: -p subxt-lightclient --no-default-features --features web -- -D warnings
+
+      - name: Cargo clippy for web subxt
+        uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: -p subxt-lightclient --no-default-features --features web -- -D warnings
+
+      - name: Cargo clippy for web subxt with light-client
+        uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: -p subxt-lightclient --no-default-features --features web,unstable-light-client -- -D warnings
+
+      - name: Cargo clippy for web subxt with light-client and jsonrpsee
+        uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: -p subxt --no-default-features --features web,unstable-light-client,jsonrpsee --target wasm32-unknown-unknown   -- -D warnings
```

Comment:
Potentially this could also be trimmed down by doing osmething like on https://github.com/paritytech/subxt/pull/1409/files/c83e0fcd133d27996a6b91ef3c9e7b46e91670e4..7320c27a0ad2f5c2f67fc499c8e14db40d20a145#diff-73e17259d77e5fbef83b2bdbbe4dc40a912f807472287f7f45b77e0cbf78792dL163